core/server/models/base/index.js

```diff
@@ -665,11 +665,6 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
      * information about the request (page, limit), along with the
      * info needed for pagination (pages, total).
      *
-     * @TODO:
-     *   - this model function does return JSON O_O
-     *   - if you refactor that out, you should double check the allowed filter options
-     *   - because `toJSON` is called in here and is using the filtered options for the `findPage` function
```

Comment:
> - because `toJSON` is called in here and is using the filtered options for the `findPage` function 

Did you double check that?

Reply:
The options that are passed to the `toJSON` method now are even less filtered than they used to (which should not be a problem). For actual filtering of the data that is being fetched, still happens [here](https://github.com/TryGhost/Ghost/pull/9899/files/3535a20cb6496e9d684b16bf2ba10650c3902134#diff-f8d02ad12b13d44b912bbf46cf7193b6L688) with the options that are allowd in e.g [here](https://github.com/TryGhost/Ghost/blob/master/core/server/models/post.js#L563). Maybe I'm missing some specific usecase?

Reply:
> The options that are passed to the toJSON method now are even less filtered than they used to

Yes that is true, because we call toJSON in the api controller - so the options are not filtered from `findPage`.

The thing i am interested in is, are there allowed options for `findPage`, which can be removed now, because `findPage` and `toJSON` are no longer dependent on each other e.g. `absolute_urls` option is allowed for `findPage`.